src/test/java/org/openmrs/contrib/qaframework/automation/ClinicalVisitSteps.java

```diff
@@ -233,9 +241,16 @@ public void systemloadsAttachmentsPage() {
 		assertTrue(textExists("Attachments"));
 	}
 
-	@When("a user attaches patient supporting file")
-	public void addSupportingFile() {
-		attachmentsPage.setFileUrl("/home/opensource/Documents/Form.pdf");
+	@And("a user attaches patient supporting file")
+	public void addSupportingFile() throws IOException {
+		File tempFile = File.createTempFile("file", ".pdf");
```

Comment:
@ibacher this is the line 246 that is throwing the nullpointer exception. I will keep trying on my side too. 
Thanks.